### DISCUSSION TITLE
Integrate with XO

### DISCUSSION THREAD

**Discussion by UltiRequiem** [neoclide/coc.nvim#3615] [Extension request]
It would be cool if there was an extension for:

https://github.com/xojs/xo

XO is basically an eslint wrapper so, I'm not sure, but shouldn't be so difficult to implement.

**Comment by yaegassy** [comment]
Hi, @UltiRequiem 

I found a VSCode extension called `vscode-linter-xo`. It seems to be an extension made by the official (xojs).

- <https://github.com/xojs/vscode-linter-xo>

You can use this as a reference to create your own coc extension.
